### PR TITLE
feat(coredns-custom): add hosts override

### PR DESCRIPTION
**Pull request by MichaelPatsula**
AAW can't resolve Cloud Main hosts. This PR adds an entry to resolve the Cloud Main Gitlab & Alertmanager hosts to the necessary ingress gateway IP for HTTPs traffic.

### CODE REVIEW

File: kube-system/coredns-custom.jsonnet [hunk 1]
```diff
@@ -33,12 +33,12 @@ local rewrite(subdomain, gateway, domain=aawDomain) = "rewrite name %(subdomain)
   data: {
     'hosts.override': |||
```
Comment (justbert) [inline kube-system/coredns-custom.jsonnet:34]: We'll just have to make sure that the `cloudmain.override` gets removed.

**Comment by justbert** [comment]
Will these changes impact this: https://github.com/StatCan/aaw-argocd-manifests/pull/260/files

File: kube-system/coredns-custom.jsonnet (hunk 1 above)
Comment (MichaelPatsula) [reply kube-system/coredns-custom.jsonnet:34]: I pulled the latest changes and removed the cloudmain.override under the data field and its related variables. Do you think host.override is better or do you think I should change it back? What is considered Cloud Main?

Comment (justbert) [reply kube-system/coredns-custom.jsonnet:34]: I think it's fine! :)

**Review by justbert — APPROVED** [review]
I think this looks good!

**Pull request merged**